docs/cave/d2d-gis-shapefiles.md

```diff
@@ -36,26 +40,25 @@ Use the Apply button to save any changes you make to the default attribute setti
+You have the ability to highlight specific areas of the displayed map or map product image, and to hide other areas. These functionalities are executed through the **Attributes dialog**, commonly referred to as the "Attributes Table" because of its tabular style. To access the **Attributes dialog**, right click and hold mouse button 3 on the Map ID in the Legend area to open the pop-up menu.
 
 From the pop-up menu select the **Display Attributes** option to open the **Attributes** dialog.
 
 ![image](../images/gisattr.png)
 
 The Attributes dialog includes two menus, Annotation and Data, which allow you to control highlighting and visibility for both the displayed images and the information contained in the Attributes dialog. For each row of information on the Attributes dialog there is an associated map/map product image displayed on the Main Display Pane.
 
-
 ### Highlighting
 
 #### Highlighting Selected Areas
 
-To place a highlight around a selected area of the GIS display image and highlight the corresponding row on the Attributes dialog, click and hold B3 on the row you want to select. Check the Highlighted checkbox in the pop-up menu to activate highlighting. Multiple rows can be selected using Ctrl + B1 to select each row, which will highlight the selected row in blue. Then click and hold B3 on one of the selected rows; this will turn all selected blue rows dark gray  and open the pop-up menu. After checking the Highlighted checkbox all selected rows will be highlighted yellow and a highlighted border surrounding all the areas on the map corresponding to those selected rows will be highlighted with the color setup in the Preferences dialog box.
+To place a highlight around a selected area of the GIS display image and highlight the corresponding row on the Attributes dialog, right click and hold B3 on the row you want to select. Multiple rows can be selected using Ctrl + B1 to select each row, which will highlight the selected row in blue. Then click and hold B3 on one of the selected rows and check the Highlighted checkbox in the pop-up menu to activate highlighting. After checking the Highlighted checkbox all selected rows will be highlighted yellow and a highlighted border surrounding all the areas on the map corresponding to those selected rows will be highlighted with the color setup in the Preferences dialog box.
 
 #### Unhighlighting Selected Areas
 
 To unhighlight selected rows and associated highlighted borders around the areas of the image, click and hold B3 on the row or select multiple rows via Ctrl + B1 and then click and hold B3 to open the pop-up menu and uncheck the Highlighted checkbox.
 
-#### ◦ Unhighlighting All Areas
+#### Unhighlighting All Areas
 
 To remove all highlighted borders around the areas of the image and remove all yellow highlighted rows on the Attributes dialog, select the Clear Highlights option under the Annotation menu.
 
```

Comment:
The very next section is using the `>` syntax to make a note, but I think we should use the `!!! note "text"` for consistency with the rest of our documentation.